redbeams/src/main/java/com/sequenceiq/redbeams/service/dbserverconfig/DatabaseServerConfigService.java

```diff
@@ -320,6 +325,22 @@ boolean validateDatabaseName(String databaseName) {
 
     @Override
     public AuthorizationResourceType getResourceType() {
-        return AuthorizationResourceType.DATALAKE;
+        return AuthorizationResourceType.DATABASE_SERVER;
+    }
+
+    @Override
+    public String getResourceCrnByResourceName(String resourceName) {
+        return repository.findByName(resourceName).get().getResourceCrn().toString();
+    }
+
+    @Override
+    public List<String> getResourceCrnListByResourceNameList(List<String> resourceNames) {
+        return repository.findByNameIn(resourceNames).stream()
```

Comment:
Can you get just the crn from the database?

Reply:
I made repository methods for these as suggested